### PR TITLE
fixed prettier formatting

### PR DESCRIPTION
**Pull request by tumms2021389**
- Fixed Prettier in the code
- Update ESlint rules to match with react-sdk-components
- Eslint formatting issue fix
- Webpack build content issue fix

### CODE REVIEW

File: __mocks__/DetailsFields.jsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
+import React, { createElement, isValidElement } from 'react';
```
Comment (ghoshArnab) [inline __mocks__/DetailsFields.jsx:1]: Ok if only extension changed to .jsx

File: __mocks__/react_pconnect.jsx [hunk 2]
```diff
@@ -14,36 +14,51 @@ let detailsIndex = 0;
 let detailsRegionIndex = 0;
 
 const createPConnectComponent = () => {
-  const formComponents = [TextInput, TextInput,TextInput, Phone, AutoComplete, Email];
+  const formComponents = [TextInput, TextInput, TextInput, Phone, AutoComplete, Email];
   const detailsComponents = [TextInput, TextInput, TextInput, Email, Phone, Currency, RadioButtons, Operator];
-  const detailsRegionComponents = [TextInput, TextInput, TextInput, Email, Phone, Currency, RadioButtons, TextInput, TextInput, TextInput, Email, Phone, Currency, Operator];
+  const detailsRegionComponents = [
+    TextInput,
```
Comment (ghoshArnab) [inline __mocks__/react_pconnect.jsx:20]: Components are repeated multiple times

File: constellation-comp-config/tsconfig.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
     "sourceMap": true,
     "module": "es2020",
```
Comment (ghoshArnab) [inline constellation-comp-config/tsconfig.json:7]: Config needs to be updated similar to root tsconfig

**Review by ghoshArnab — CHANGES_REQUESTED** [review]
Please check comments

File: __mocks__/react_pconnect.jsx (hunk 2 above)
Comment (vishalshrm539) [reply __mocks__/react_pconnect.jsx:20]: @ghoshArnab This is intentional as part of moacking createPConnectComponent API.

**Review by paulgagnon-pega — DISMISSED** [review]

File: constellation-comp-config/tsconfig.json (hunk 3 above)
Comment (4manasa) [reply constellation-comp-config/tsconfig.json]: > Config needs to be updated similar to root tsconfig

updated

**Review by ghoshArnab — APPROVED** [review]

**Pull request merged**